src/Molecules/UserSelect/User/User.jsx

```diff
@@ -35,7 +45,11 @@ export const User = ({
       <FlexBox component={selected ? Check : null} width={45} size={25} marginLeft={1} />
       <FlexBox component={avatar ? Avatar : null} alt={avatar} src={avatar} classes={avatar ? avatarClasses : null} />
       <FlexBox flexGrow={1} component={Typo}>{children}</FlexBox>
-      <FlexBox component={status ? Circle : null} color={`${status}.main`} width={60} size={25} title={status} />
+      <Tooltip title={tooltip} placement="bottom" classes={tooltipClasses}>
```

Comment:
It seems like this breaks the original styles of the Tootlip styles (the background is grey instead of black)